module/metrics/verification.go

```diff
@@ -145,6 +147,17 @@ func NewVerificationCollector(tracer module.Tracer, registerer prometheus.Regist
 		Help:      "total number of chunk data packs requested by fetcher engine",
 	})
 
+	maxChunkDataPackRequestAttemptForNextUnsealedHeight := prometheus.NewGauge(prometheus.GaugeOpts{
+		Name:      "next_unsealed_height_max_chunk_data_pack_request_attempt_times",
+		Namespace: namespaceVerification,
+		Subsystem: subsystemRequesterEngine,
+		// an indicator for when execution nodes is unresponsive to chunk data pack requests,
+		// in which case verification node will keep requesting the chunk data pack, and this
+		// metrics number will go up.
+		Help: "among all the pending chunk data packs requested by the requester engine for the next unsealed height the maximum number of times a" +
+			" certain chunk data pack was requested",
```

Comment:
```suggestion
		Help: "the maximum number of times the chunk data pack for the next unsealed height was requested by the requester engine",
```

Reply:
@zhangchiqing my apologies, just saw this comment after merging, will apply it on another PR of mine.